packages/openapi-v3/src/controller-spec.ts

```diff
@@ -89,16 +90,55 @@ function resolveControllerSpec(constructor: Function): ControllerSpec {
       endpointName = `${fullMethodName} (${verb} ${path})`;
     }
 
+    const defaultResponse = {
+      '200': {
+        description: `Return value of ${constructor.name}.${op}`,
+      },
+    };
+
     let operationSpec = endpoint.spec;
     if (!operationSpec) {
       // The operation was defined via @operation(verb, path) with no spec
       operationSpec = {
-        responses: {},
+        responses: defaultResponse,
       };
       endpoint.spec = operationSpec;
     }
     debug('  operation for method %s: %j', op, endpoint);
 
+    debug('  spec responses for method %s: %o', op, operationSpec.responses);
+
+    const TS_TYPE_KEY = 'x-ts-type';
+
+    for (const code in operationSpec.responses) {
+      for (const c in operationSpec.responses[code].content) {
```

Comment:
IIRC, `content` is an optional property. Please add a test to verify what happens when the OAI spec does not provide any value for `content`, and fix the implementation if needed.

Reply:
`content` is an optional property and definitely took me a minute to realize what was happening here since I coded it. 

The `for (const c in operationSpec.responses[code].content)` loop is only entered if the `operationSpec.responses[code]` object had a `content` property. If the property is not defined, the loop is never entered as there are no properties on `undefined` -- making the implementation correct and sound. 

I also verified this behaviour on this page: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in

By changing the code to be as follows: 

```
var string1 = "";
var object1 = {a: 1, b: 2, c: 3};
// var object1 = {a: 1, b: 2, c: 3, d: {x:1, y:2}}; 

for (var property1 in object1.d) {
  console.log('in loop')
  string1 = string1 + property1;
}

console.log(string1);
// expected output: "123"
```

The above will log `""` as is and uncommenting line 3 should give `"xy"` after 2x `in loop`